src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -266,6 +266,7 @@
             <argument type="service" id="security.access.decision_manager" />
             <argument type="service" id="security.access_map" />
             <argument type="service" id="security.authentication.manager" />
+            <argument type="service" id="security.authorization_checker" on-invalid="null"/>
```

Comment:
I'm not entirely sure the `on-invalid="null"` is needed. In case of an additional argument missing (old component version), it would just pass an extra argument to the class, which is allowed by php. So I think this can be a normal required argument.

Reply:
You're right.